packages/core/src/libraries/social.ts

```diff
@@ -65,7 +70,8 @@ export const createSocialLibrary = (queries: Queries, connectorLibrary: Connecto
   const getUserInfoByAuthCode = async (
     connectorId: string,
     data: unknown,
-    getConnectorSession: GetSession
+    getConnectorSession: GetSession,
+    storage: { set: SetStorageValue; get: GetStorageValue }
```

Comment:
Should these 2 be optional params?

Reply:
Since there is only one `getUserInfo` call in @logto/core and all input parameters will be passed in, there is no big difference for whether this parameter is required or optional. For the same reason that we make `getConnectorSession` as a required parameter, we do not need to check the parameter existence and type when we try to use them in connectors' `getUserInfo` method, which can bring convenience.